site/src/components/scroll.md

```diff
@@ -65,3 +65,134 @@ Force an elements overflow to be horizontally scrollable by using `.scroll-verti
+
+```html
+<section id="snap-scroll-1" class="width-full width-500-tablet display-none-widescreen" data-snap-scroll="first-one">
+	<div class="scroll-horizontal scroll-snap-container display-flex padding-bottom-xs" data-snap-scroll-slides>
+		<article class="padding-sm border scroll-snap-item" id="one" data-snap-scroll-slide="first-one-1">
+			<h3 class="font-size-lg">1</h3>
+			<p>Description text is here and goes on for a little while.</p>
+		</article>
+		<article class="padding-sm border scroll-snap-item" id="two" data-snap-scroll-slide="first-one-2">
+			<h3 class="font-size-lg">2</h3>
+			<p>Description text is here and goes on for a little while.</p>
+		</article>
+		<article class="padding-sm border scroll-snap-item" id="three" data-snap-scroll-slide="first-one-3">
+			<h3 class="font-size-lg">3</h3>
+			<p>Description text is here and goes on for a little while.</p>
+		</article>
+		<article class="padding-sm border scroll-snap-item" id="four" data-snap-scroll-slide="first-one-4">
+			<h3 class="font-size-lg">4</h3>
+			<p>Description text is here and goes on for a little while.</p>
+		</article>
+		<article class="padding-sm border scroll-snap-item" id="five" data-snap-scroll-slide="first-one-5">
+			<h3 class="font-size-lg">5</h3>
+			<p>Description text is here and goes on for a little while.</p>
+		</article>
+		<article class="padding-sm border scroll-snap-item" id="six" data-snap-scroll-slide="first-one-6">
+			<h3 class="font-size-lg">6</h3>
+			<p>Description text is here and goes on for a little while.</p>
+		</article>
+		<article class="padding-sm border scroll-snap-item" id="seven" data-snap-scroll-slide="first-one-7">
+			<h3 class="font-size-lg">7</h3>
+			<p>
+				Description text is here and goes on for a little while. <br />
+				Even more text.
+			</p>
+		</article>
+	</div>
+	<nav class="pagination" role="navigation" aria-label="pagination" data-snap-scroll-nav="first-one">
+    <ul class="pagination-list">
+        <li>
+            <a href="#first-one-1" class="pagination-link is-current" data-snap-scroll-nav-item="first-one-1">1</a>
+        </li>
+        <li>
+            <a href="#first-one-2" class="pagination-link" data-snap-scroll-nav-item="first-one-2">2</a>
+        </li>
+        <li>
+            <a href="#first-one-3" class="pagination-link" data-snap-scroll-nav-item="first-one-3">3</a>
+        </li>
+        <li>
+            <a href="#first-one-4" class="pagination-link" data-snap-scroll-nav-item="first-one-4">4</a>
+        </li>
+        <li>
+            <a href="#first-one-5" class="pagination-link" data-snap-scroll-nav-item="first-one-5">5</a>
+        </li>
+        <li>
+            <a href="#first-one-6" class="pagination-link" data-snap-scroll-nav-item="first-one-6">6</a>
+        </li>
+        <li>
+            <a href="#first-one-7" class="pagination-link" data-snap-scroll-nav-item="first-one-7">7</a>
+        </li>
+	</nav>
+</section>
+```
+
+A second example
```

Comment:
This second example can stay around for testing purposes until this PR is nearly ready, but then we can remove it. It was just a sensibility check that our code functions well enough with pages that may contain multiple such elements. It may be confusing to the develop reading this documentation though. (You could also just comment it out for easier testing down the line)

Reply:
removed it